src/filefinder.cpp

```diff
@@ -212,6 +220,14 @@ namespace {
 			if (!ret_rtp.empty()) { return ret_rtp; }
 		}
 
+		if (!disable_rtp) {
+			std::string msg = "Cannot find: %s/%s. " + std::string(search_paths.empty() ?
+				"Install RTP %d to resolve this warning." : "RTP %d was probably not installed correctly.");
+
+			Output::Warning(msg.c_str(),
+				dir.c_str(), name.c_str(), Player::IsRPG2k() ? 2000 : 2003);
+		}
+
 		Output::Debug("Cannot find: %s/%s (%s)", dir.c_str(), name.c_str(),
```

Comment:
This one is redundant, as we have dedicated warning vs. debug messages for both now.